README.md

```diff
@@ -69,44 +72,72 @@ uswds-0.13.1/
+
+Next, you will want to import these assets into your HTML. Below is an example of the code for our example projects `index.html` file:
 
 ```html
-<script src="/path/to/your/assets/js/lib/uswds.min.js"></script>
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>My Example Project</title>
+  <link rel="stylesheet" href="assets/uswds-0.13.1/css/uswds.min.css">
+</head>
+<body>
+  
+  <script src="assets/uswds-0.13.1/js/uswds.min.js"></script>
+</body>
+</html>
 ```
 
 We offer both files, the CSS and the JavaScript, in two versions — a minified
-version, and an un-minified one. (In the examples above, we are using the minified
-files.) Use the minified files in a production environment or to reduce the
-file size of your downloaded assets. And the un-minified files are better if you
-are in a development environment or would like to debug the CSS or JavaScript
-assets in the browser.
+version, and an un-minified one. (In the examples above, we are using the minified files.) Use the minified files in a production environment or to reduce the file size of your downloaded assets. And the un-minified files are better if you are in a development environment or would like to debug the CSS or JavaScript assets in the browser.
 
-This version of the Standards includes jQuery version `2.2.0` bundled within the
-JavaScript file. Please make sure that you’re not including any other version
-of jQuery on your page.
+**NOTE:** This version of the Standards includes jQuery version `2.2.0` bundled within the JavaScript file. Please make sure that you’re not including any other version of jQuery on your page.
 
-And that’s it — you should be set to use the Standards.
+And that’s it — you should now be able to copy our code samples into our `index.html` and start using the Standards.
 
 ### Install using npm
 
-If you have `node` installed on your machine, you can use npm to install the Standards. Add `uswds`
-to your project’s `package.json` as a dependency:
+`npm` is a package manager for Node.js based projects. The Draft U.S. Web Design Standards maintains a [`uswds` package](https://www.npmjs.com/package/uswds) for you to utilize both the pre-compiled and compiled files on your project.
+
+To use this method, you will need to have Node/npm installed on your local development environment. Below is a link to find the install method that coincides with your operating system:
+
+1. Node v4.2.3+, [Installation guides](https://nodejs.org/en/download/)
+
+Now let's check to make sure you have installed it correctly:
+
+```shell
+$ npm -v
+3.10.8 # This line may vary depending on what version of Node you've installed.
+```
+
+**Windows installation:** Team Treehouse has a tutorial on how to install Node.js/npm on your Windows machine: [http://blog.teamtreehouse.com/install-node-js-npm-windows](http://blog.teamtreehouse.com/install-node-js-npm-windows)
```

Comment:
Agreed. This phrasing makes Windows sound like an edge-case, which I do not think is the intent here.